### PR TITLE
Migrate to JDK 11 and Java 11 syntax [part 2]

### PR DESCRIPTION
**Pull request by armiol**
This is a second part of #698.

It completes the migration to `var` syntax. Also, deals with some warnings and inconsistencies in test naming.

Lastly, it updates the `README.md` to reflect the recent changes made to the repository code.

The version is set to `2.0.0-SNAPSHOT.79`.

### CODE REVIEW

File: base/src/test/java/io/spine/base/IdentifierTest.java [hunk 1]
```diff
@@ -368,23 +368,23 @@ void ofAny() {
...
+        var nestedMessageString = StringValue.of(nestedString);
+        var idToConvert = SeveralFieldsId.newBuilder()
                 .setString(outerString)
                 .setNumber(number)
                 .setMessage(nestedMessageString)
                 .build();
 
-        String expected =
+        var expected =
                 "string=\"" + outerString + '\"' +
```
Comment (dmdashenkov) [inline base/src/test/java/io/spine/base/IdentifierTest.java:383]: I think, this code can be moved to the previous line.

File: base/src/test/java/io/spine/validate/option/FieldValidatingOptionTest.java [hunk 2]
```diff
@@ -203,18 +197,18 @@ public String errorMessage(FieldContext field) {
...
-                                                       .setTypeName(containingMessage.declaration()
-                                                                                     .name()
-                                                                                     .value())
-                                                       .setMsgFormat(errorMessage(context))
-                                                       .build())
+                                .setFieldPath(context.fieldPath())
+                                .setTypeName(containingMessage.declaration()
+                                                              .name()
+                                                              .value())
+                                .setMsgFormat(errorMessage(context))
```
Comment (dmdashenkov) [inline base/src/test/java/io/spine/validate/option/FieldValidatingOptionTest.java:210]: Please extract repeated operation results out of the stream.

**Review by dmdashenkov — APPROVED** [review]
LGTM with a couple of comments.

File: testlib/src/test/java/io/spine/testing/ReflectiveBuilderTest.java [hunk 3]
```diff
@@ -35,13 +35,13 @@
...
-@DisplayName("ReflectiveBuilder should")
+@DisplayName("`ReflectiveBuilder` should")
 class ReflectiveBuilderTest {
 
     @Test
     @DisplayName("have the result class")
     void resultClass() {
-        ReflectiveBuilder<Any> builder =
+        var builder =
                 new DummyBuilder().setResultClass(Any.class);
```
Comment (alexander-yevsyukov) [inline testlib/src/test/java/io/spine/testing/ReflectiveBuilderTest.java:45]: Please see if this can fit into the previous line.

**Review by alexander-yevsyukov — APPROVED** [review]
LGTM, with one comment.

**Pull request merged**